### DISCUSSION TITLE
Strapi v5 Compatible Plugin

### DISCUSSION THREAD

**Discussion by BayBreezy** [alexzaganelli/strapi-plugin-email-designer#146] [Show and tell]
Hello,

I created a Strapi 5 version of this plugin. It can be found here: https://github.com/BayBreezy/strapi-plugin-email-designer-5

I would love some feedback. I am just learning React. Thought it would be good practice to use this plugin to learn some more.

**Comment by creazy231** [comment]
Why does people always need to create „new“ packages from existing ones instead of commit to existing ones?
Don’t get me wrong, appreciate the work on this but what stops you from continuing to THIS repository?

**Comment by BayBreezy** [comment]
Well, The structure of how plugins are now in strapi was the 1st thing that deterred me from just creating a new branch and try to make it work.
Second was, I am new to React.. like just started last week new. Dont think I should be taking over something that so many ppl are relying on.

**Reply by jouleetech** [reply]
Hey, i thank you so much, that you got it refactored for strapi v5, can we exchange emails ive been working on this last week 2 and now lets give you the credit of the designer refactoring.

**Reply by BayBreezy** [reply]
Hey 👋.  
behon.baker@yahoo.com is the email. Have you gotten a chance to look at the source code? Is there anything that can be improved? Like I said, I just started learning react recently and I am not sure if there is anything wrong that I am doing. Would be nice if an experienced rract person could take a look

**Reply by jouleetech** [reply]
wrote you an email :) i would say thank you, for your effort here!

**Comment by hampton1122** [comment]
@BayBreezy and @begservice Thanks for taking this on. This plugin is the only reason why I have not migrated to Version 5. Please let us know when version 5 is ready for production.

**Comment by BayBreezy** [comment]
@hampton1122 I will be working on this next weekend. 

I plan to add pagination to the template table as well as adding a table that track the changes made to a template.

Thanks for waiting 🙏🏽. We will get this done.

I built this because I wanted to use strapiv5 too but needed this plugin. Thankfully the plugin sdk released by strapi allowed a noob like me to create a plugin LOLOL

**Reply by jouleetech** [reply]
Man dont play your work here down, haha you did a good job, Thank you for your work here, i hope at least you will stick to it to support it! Its important to a lot of people to keep this running!
